src/containers/NFT/NFTHeader/index.js

```diff
@@ -0,0 +1,214 @@
+            <td className="col1">Transfer Fee</td>
+            <td className="col2">{transferFee}</td>
+          </tr>
+        </tbody>
+      </table>
+    )
+  }
+
+  const renderSettings = () => {
+    const { flags } = data
+
+    const burnable = flags?.includes('lsfBurnable') ? 'enabled' : 'disabled'
+    const onlyXRP = flags?.includes('lsfOnlyXRP') ? 'enabled' : 'disabled'
+    const trustLine = flags?.includes('lsfTrustLine') ? 'enabled' : 'disabled'
+    const transferable = flags?.includes('lsfTransferable')
+      ? 'enabled'
+      : 'disabled'
+    return (
+      <table className="token-table">
+        <tbody>
+          <tr className="row">
+            <td className="col1">Burnable</td>
+            <td className="col2">{burnable}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Only XRP</td>
+            <td className="col2">{onlyXRP}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Trust Line</td>
+            <td className="col2">{trustLine}</td>
+          </tr>
+          <tr className="row">
+            <td className="col1">Transferable</td>
+            <td className="col2">{transferable}</td>
+          </tr>
+        </tbody>
+      </table>
+    )
+  }
+
+  const renderHeaderContent = () => {
+    const { issuer } = data
+    const abbrvIssuer =
+      issuer?.length > 23 ? issuer?.slice(0, 23).concat('...') : issuer
+    return (
+      <div className="section nft-header-container">
+        <div className="nft-info-container">
+          <div className="values">
+            <div className="title">{t('issuer_address')}</div>
+            <div className="value">
+              <Link className="value-content" to={`/accounts/${issuer}`}>
+                {abbrvIssuer}
+              </Link>
+              <CopyToClipboard className="copy" text={issuer} />
+            </div>
+          </div>
+        </div>
+        <div className="nft-bottom-container">
+          <div className="details">
+            <div className="title">{t('details')}</div>
+            {renderDetails()}
+          </div>
+          <div className="settings">
+            <div className="title">{t('settings')}</div>
+            {renderSettings()}
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className="nft-token-header">
+      <div className="section">
+        {!loading && data && (
+          <div className="nft-box-header">
+            <div className="token-title">
+              NFT ID
+              <div className="token-type">
+                <div className="subscript">NFT</div>
+              </div>
+            </div>
+            <div
+              className="title-content"
+              onMouseOver={(e) => showTooltip(e, { tokenId })}
+              onFocus={() => {}}
+              onMouseLeave={hideTooltip}
+            >
+              {tokenId}
+            </div>
+          </div>
+        )}
```

Comment:
you can render all of this safely even without `data` and even if still `loading`

Reply:
you're still not rendering this if the component is loading. i don't see why?